src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -163,6 +164,11 @@ def do_value_profile():
 
 def do_fork():
   """Return whether or not to do fork mode."""
+  # TODO(metzman): Find a workaround for Windows command line limit before
```

Comment:
I have a theory that using subset allows fork mode to work. Might be worth looking into.

Reply:
filed https://github.com/google/clusterfuzz/issues/267